setup.sh

```diff
@@ -19,7 +19,7 @@ tar xvf wasi-sdk-11.0-"${system}".tar.gz
 # binaryen
 git clone --branch wasi-stub --single-branch https://github.com/near/binaryen
 cd binaryen
-cmake . && make -j
+cmake . && make -j 8
```

Comment:
Is there developers with less than 4 core 8 threads? If we need to consider those developers, maybe simply drop `-j`. `make` is slower than `make -j 8` or `make -j`, but it'll work on slow/less RAM computers. `setup.sh` is a one time thing, slower is okay.

```suggestion
cmake . && make
```